test/core/format/RawAddress.spec.ts

```diff
@@ -369,11 +325,11 @@ describe('address', () => {
             ];
 
             const Addresses = [
-                'NDIPRQMB3HT7A6ZKV7HOHJQM7JHX6H3FN7DBVFHB',
-                'NC65QJI4OWTUFJNQ2IDVOMUTE7IDI2EGEFP5QME7',
-                'NCBC4VAQBVSB4J5J2PTFM7OUY5CYDL33VW7RKW7K',
-                'NBLW3CQPBGPCFAXG4XM5GDEVLPESCPDPFO7NUCOM',
-                'NA5RDU36TKBTW4KVSSPD7PT5YTUMD7OIJFMMIEEQ',
+                'MDIPRQMB3HT7A6ZKV7HOHJQM7JHX6H3FN5YHHZMD',
```

Comment:
Maybe you can load the test cases directly from the vector files. I've added that to Java sdk.

Reply:
Good point, but the size of the test vector json files is a concern. To keep all unit tests run offline, need to copy the files to the project.